newsletter.md

```diff
@@ -0,0 +1,46 @@
+# ZK-newsletter for 2023-02-08
+Summary of the most important things, which happened in Zero Knowledge world, in the previous week.
+
+### New articles
+* [Understanding Zero Knowledge Proofs — A Beginner’s Guide](https://medium.com/@info.marcinjedral/understanding-zero-knowledge-proofs-a-beginners-guide-a78bc5bf5644) - article introducing ZKP by Marcin Jedral. (01.02.2023) 
+* [Zeroing in on Security: The Concept of Zero-Knowledge Proofs](https://jaylanthedev.medium.com/zeroing-in-on-security-the-concept-of-zero-knowledge-proofs-ac67c3cb9825) - article by Jaylan Snelson. (02.02.2023)
+* [Zero-Knowledge Proof and ZKP Projects: Exploring the Future of Privacy and Security](https://earlyminter.com/zero-knowledge-proof-and-zkp-projects-exploring-the-future-of-privacy-and-security/) - article by Matheus Villas Boas. (03.02.2023)
+* [Wasm0 (ZKWasm): A New Blockchain Environment for Mass Adoption](https://medium.com/@0xwasm/wasm0-zkwasm-a-new-blockchain-environment-for-mass-adoption-15aab385a7ee) - article by ZKWasm. (03.02.2023)
+* [Zero Knowledge Proofs (ZKPs) and ZK-Rollups in Crypto: Everything You Need to Know](https://fxdailyreport.com/zero-knowledge-proofs-zkps-zk-rollups/) - article diving into ZKPs and Zk-Rollups by Daffa Zaky (03.02.2023)
+* [Zero-Knowledge Rollup — Part 1](https://medium.com/integraate/zero-knowledge-rollup-part-1-7584327e19d) - article about ZKRollups with code examples by Kushagra Jindal. (05.02.2023)
+* [Let’s dissect a zkSNARK! (part 1)](https://medium.com/@yujiangtham/lets-dissect-a-zksnark-part-1-a82fc092f58a) - article explainig ZKSnarks by Yu Jiang Tham. (06.02.2023)
+* [Polygon’s Zero Knowledge Odyssey Part I](https://medium.com/@sanilshah1999/polygons-zero-knowledge-odyssey-part-1-6-3abb67cb5da8), [Part II](https://medium.com/@sanilshah1999/polygons-zero-knowledge-odyssey-part-2-89ad9f6533f0), [Part III](https://medium.com/@sanilshah1999/polygons-zero-knowledge-odyssey-part-3-cfb9c8d81cf1) - 3 parts of 6-part series of articles, they take a closer look at some of the key acquisitions and developments that have helped shape the future of Polygon using the power of Zero Knowledge (ZK). (01-07.02.2023)
+
+### Layer 2 projects
+* [Dymension](https://medium.com/@dymensionXYZ/dymension-35-c-4dc8ac1c987c) announced that their public testnet is coming in *Q2 2023*. (1.02.2023)
+* [Announcment of the Starknet](https://twitter.com/StarkWareLtd/status/1622197933025251328) - StarkWare has announced that it will be making its software open-source. (05.02.2023)
+
+### YouTube Clips
+* [Common Zero-Knowledge Proof Vulnerabilities](https://www.youtube.com/watch?v=1RQSwj8h8rM) - lecture by D-Square. (02.02.2023)
+* [Lecture 2: Overview of Modern SNARK Constructions](https://www.youtube.com/watch?v=bGEXYpt3sj0) - lecture from Zero Knowledge Proofs MOOC Spring '23. (02.02.2023)
```

Comment:
It's better if the name doesn't start with `Lecture` or `Lesson`. Prepend those with the name of the course/yt channel.